src/main/java/org/jenkinsci/plugins/tokenmacro/impl/LogRegExMacro.java

```diff
@@ -50,15 +54,21 @@ public String readLogFile(File file) throws IOException
         // Assume default encoding and text files
         String line;
         Pattern pattern = Pattern.compile(regex);
-        BufferedReader reader = new BufferedReader(new FileReader(file));
-        while ((line = reader.readLine()) != null)
-        {
-            Matcher matcher = pattern.matcher(line);
-            if (matcher.find())
+        
+        BufferedReader reader = new BufferedReader(
+                new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8));
```

Comment:
Yes, of course. I'll revert the stuff to default encodings and add an optional parameter. Since it's possible to configure any encoding in Jenkins, any other option is wrong